google/cloud/security/scanner/scanner.py

```diff
@@ -340,13 +352,9 @@ def _build_scan_summary(all_violations, total_resources):
             resource_summaries[resource_type][
                 'violations'][violation.resource_id] = 0
 
-        # Make sure to count each member violation as a separate one.
-        # TODO: Refactor as the loop is not required.
-        for _ in violation.members:
-            resource_summaries[resource_type][
-                'violations'][violation.resource_id] += 1
-
-            total_violations += 1
+        resource_summaries[resource_type][
```

Comment:
Not looking, but do we have a test to make sure these counts come out equal? If not it might be good to add one.